### PR TITLE
Deploy base app to heroku

### PR DESCRIPTION
**Pull request by clintfidel**
#### What does this PR do?
Deploy base app to heroku

#### Description of Task to be completed?
- Deploy base app to heroku and enable auto deploy

#### How should this be manually tested?
- raise pull request against base branch for auto deploy
- link to hosted app [here](https://asgard-cfh-staging.herokuapp.com/)

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
#157801280

### CODE REVIEW

**Review by fortunee — COMMENTED** [review]
Hi @clintfidel  we have conflicts here now, please let me know if you need help fixing it. 😄

Also, let's have a link to the Heroku app on your PR description. Thanks

**Comment by clintfidel** [comment]
I have resolved the merge conflict @fortunee, please review.

File: app.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "asgard-cfh",
+    "scripts": {},
```
Comment (fortunee) [inline app.json]: Everything here except for `name` seems redundant as they hold no actual value, please get rid of them ASAP. Good job @clintfidel 😄

**Review by fortunee — COMMENTED** [review]
Hi @clintfidel Please review the comment I left on this PR. Thanks

File: app.json (hunk 1 above)
Comment (fortunee) [reply app.json]: Also, link to the hosted app isn't correct. Please fix it.

**Comment by clintfidel** [comment]
@fortunee Thanks for the feedback, I have addressesd your comments.

**Pull request merged**